client/containers/App/paths.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import {
+	About,
+	AdminDashboard,
+	CommunityCreate,
+	Dashboard,
+	DashboardMembers,
+	DashboardOverview,
+	Explore,
+	Landing,
+	Login,
+	NoMatch,
+	Page,
+	PasswordReset,
+	Pricing,
+	Privacy,
+	Pub,
+	Search,
+	Signup,
+	Terms,
+	User,
+	UserCreate,
+} from 'containers';
+
+export default (viewData, locationData, chunkName) => {
```

Comment:
This feels good. My only suggestion is to note that this function doesn't depend on `viewData` in any meaningful way...it just applies it to the `activeComponent` and sends that back to the caller, and means that we have to write `<Component {...viewData} />` over and over here. Better perhaps to do e.g.:

```js
const paths = {
    About: {
         Component: About,
         // ...
    },
    // ...
}
```
and then in `App.js` just render `<Component {...viewData} />` the one time.

Reply:
Definitely - good catch. A copy-paste holdover that didn't get enough thought. I'll make that change.